Pokégen.PkHex/Services/PokemonService.cs

```diff
@@ -40,15 +42,11 @@ public async Task<PKM> GetPokemonFromFormFileAsync(IFormFile file, SupportedGame
 	public Task<PKM> GetPokemonFromShowdown(string showdownSet, SupportedGame game)
 	{
 		var set = new ShowdownSet(showdownSet);
-		var template = set.GetTemplate();
```

Comment:
Do not remove the error handling in here as its needed for ASP.Net to properly recognize this an a non 500 exception